modulefiles/ufs_gaea.intel.lua

```diff
@@ -22,11 +16,13 @@ load(pathJoin("stack-cray-mpich", stack_cray_mpich_ver))
 stack_python_ver=os.getenv("stack_python_ver") or "3.10.8"
 load(pathJoin("stack-python", stack_python_ver))
 
+cmake_ver=os.getenv("cmake_ver") or "3.23.1"
+load(pathJoin("cmake", cmake_ver))
+
 load("ufs_common")
-load("nccmp/1.9.0.1")
 
-unload("darshan-runtime")
-unload("cray-libsci")
```

Comment:
Why are these removed?

Reply:
I know we need to unload `cray-libsci`. The intel and cray libraries contain routines of the same name and using the wrong one can cause a program to freeze while finalizing libraries (ie. during atexit handlers). As for darshan-runtime, I don't know what that is.

Please try putting those two lines back in.

Reply:
Adding the unload darshan and cray-libsci back to the module file seemed to work: `/gpfs/f5/ufs-ard/world-shared/for_sam/rt_126272`.
We still get these notices:
`0: NOTE from PE     0: Potential error in diag_manager_end: slp NOT available, check if output interval > runlength. Netcdf fill_values are written,`
I updated the module file in the repo.

This is from Gaea docs about darshan:
When the darshan-runtime/3.4.0 module is active with non-Intel compilers, an error is thrown at link time regarding Intel specific libraries that can't be found. This is an ongoing issue and is currently being worked on by ORNL (https://gaeadocs.rdhpcs.noaa.gov/wiki/index.php?title=Known_issues&title=Known_issues)

We should probably clean up tests/rt.sh as well, since the gaea module file is handling the module loads.  I'm not sure how much of this is needed:
```
    export LD_PRELOAD=/opt/cray/pe/gcc/12.2.0/snos/lib64/libstdc++.so.6
    module load PrgEnv-intel/8.3.3
    module load intel-classic/2023.1.0
    module load cray-mpich/8.1.25
    module load python/3.9.12
    module use /ncrc/proj/epic/spack-stack/modulefiles
    module load gcc/12.2.0
```

Reply:
> We still get these notices:
> `0: NOTE from PE     0: Potential error in diag_manager_end: slp NOT available, check if output interval > runlength. Netcdf fill_values are written,`

Those errors are okay. Actually, it's probably best to leave that in as a test of whether the diag manager is correctly filling with fill_values.